File: common/src/main/java/se/cygni/texasholdem/game/Hand.java
package se.cygni.texasholdem.game;

import java.util.List;

import se.cygni.texasholdem.game.definitions.PokerHand;

public class Hand {

    private final List<Card> cards;
    private final PokerHand pokerHand;

    public Hand(final List<Card> cards, final PokerHand pokerHand) {

        this.cards = cards;
        this.pokerHand = pokerHand;
    }

    public List<Card> getCards() {

        return cards;
    }

    public PokerHand getPokerHand() {

        return pokerHand;
    }

}

File: client-implementations/python-client/common/game/Hand.py
#!/usr/bin/env python

from common.game.definitions.PokerHand import *


class Hand(object):
    """ generated source for class Hand """
    cards = []
    pokerHand = None    # type: PokerHand
    folded = False      # type: bool

    def __init__(self, cards, pokerHand, folded = False):
        """ generated source for method __init__ """
        self.folded = folded
        self.cards = [] if folded else cards
        self.pokerHand = PokerHand.NOTHING if folded else pokerHand

    def getCards(self):
        """ generated source for method getCards """
        return self.cards

    def getPokerHand(self):
        """ generated source for method getPokerHand """
        return self.pokerHand

    def isFolded(self):
        """ generated source for method isFolded """
        return self.folded

